frontend/web/components/pages/GettingStartedPage.tsx

```diff
@@ -0,0 +1,359 @@
+import React, { FC, useEffect, useState } from 'react'
+import PageTitle from 'components/PageTitle'
+import Button from 'components/base/forms/Button'
+import loadCrisp from 'common/loadCrisp'
+import Utils from 'common/utils/utils'
+import ConfigProvider from 'common/providers/ConfigProvider'
+import Icon, { IconName } from 'components/Icon'
+import { Link } from 'react-router-dom'
+import { useGetProjectsQuery } from 'common/services/useProject'
+import AccountStore from 'common/stores/account-store'
+import classNames from 'classnames'
+import { useGetProjectFlagsQuery } from 'common/services/useProjectFlag'
+import { useGetSegmentsQuery } from 'common/services/useSegment'
+import flagsmith from 'flagsmith'
+import Tooltip from 'components/Tooltip'
+import { useGetEnvironmentsQuery } from 'common/services/useEnvironment'
+import API from 'project/api'
+type ResourcesPageType = {}
+type GettingStartedItem = {
+  duration: number
+  title: string
+  description: string
+  link: string
+  cta?: string
+  icon?: IconName
+  complete?: boolean
+  disabledMessage?: string | null
+  persistId?: string
+}
+type GettingStartedItemType = {
+  data: GettingStartedItem
+}
+
+const resources: {
+  description: string
+  image: string
+  title: string
+  url: string
+}[] = [
+  {
+    'description':
+      'A hands-on guide covering best practices, use cases and more.',
+    'image': '/static/images/welcome/featured1.png',
+    'title': 'eBook | Unlock Modern Software Development with Feature Flags',
+    url: 'https://www.flagsmith.com/ebook/flip-the-switch-on-modern-software-development-with-feature-flags?utm_source=app',
+  },
+  {
+    'description': 'Set yourself up for success with these best practices.',
+    'image': '/static/images/welcome/featured2.png',
+    'title': 'Blog Post | Feature Flag best practices',
+    url: 'https://www.flagsmith.com/blog/feature-flags-best-practices?utm_source=app',
+  },
+]
+
+const GettingStartedItem: FC<GettingStartedItemType> = ({ data }) => {
```

Comment:
To aerate the file we could move them in a dedicated folder and separate:
- GettingStartedItem
- GettingStartedPage
- items and resources in a separate data file